roles/create_vms/tasks/prepare_network.yml

```diff
@@ -1,6 +1,16 @@
 - name: Setup network
   become: true
   block:
+    - name: Check if the selected network mode is allowed
```

Comment:
Could you move this into the `validate_inventory` role that way it will be checked at the start of the execution so people don't have to wait for feedback :)

Reply:
Sure, done